README.md

```diff
@@ -1,5 +1,5 @@
 
-[![Edge](https://github.com/nginxinc/kubernetes-ingress/workflows/Edge/badge.svg)](https://github.com/nginxinc/kubernetes-ingress/actions?query=workflow%3AEdge)  [![Nightly](https://github.com/nginxinc/kubernetes-ingress/workflows/Nightly/badge.svg)](https://github.com/nginxinc/kubernetes-ingress/actions?query=workflow%3ANightly)  [![FOSSA Status](https://app.fossa.io/api/projects/custom%2B1062%2Fgithub.com%2Fnginxinc%2Fkubernetes-ingress.svg?type=shield)](https://app.fossa.io/projects/custom%2B1062%2Fgithub.com%2Fnginxinc%2Fkubernetes-ingress?ref=badge_shield)  [![License](https://img.shields.io/badge/License-Apache%202.0-blue.svg)](https://opensource.org/licenses/Apache-2.0)  [![Go Report Card](https://goreportcard.com/badge/github.com/nginxinc/kubernetes-ingress)](https://goreportcard.com/report/github.com/nginxinc/kubernetes-ingress)
+[![Edge](https://github.com/nginxinc/kubernetes-ingress/workflows/Edge/badge.svg)](https://github.com/nginxinc/kubernetes-ingress/actions?query=workflow%3AEdge)  [![Nightly](https://github.com/nginxinc/kubernetes-ingress/workflows/Nightly/badge.svg)](https://github.com/nginxinc/kubernetes-ingress/actions?query=workflow%3ANightly)  [![FOSSA Status](https://app.fossa.com/api/projects/custom%2B5618%2Fgithub.com%2Fnginxinc%2Fkubernetes-ingress.svg?type=shield)](https://app.fossa.com/projects/custom%2B5618%2Fgithub.com%2Fnginxinc%2Fkubernetes-ingress?ref=badge_shield)  [![License](https://img.shields.io/badge/License-Apache%202.0-blue.svg)](https://opensource.org/licenses/Apache-2.0)  [![Go Report Card](https://goreportcard.com/badge/github.com/nginxinc/kubernetes-ingress)](https://goreportcard.com/report/github.com/nginxinc/kubernetes-ingress)
```

Comment:
What's the second fossa badge link for (the one ending in "ref=badge_shield")? I don't know if it's doing anything.

Reply:
not sure, but that's the snippet from Fossa's page